src/global.css.js

```diff
@@ -16,10 +16,8 @@ export default createGlobalStyle`
   menu, nav, output, ruby, section, summary,
   time, mark, audio, video {
     margin: 0;
-    padding: 0;
     border: 0;
     font-size: 100%;
-    font: inherit;
```

Comment:
This was the cause of the bold font not showing up. Why would they do this to us?? lol